backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/dao/impl/QueryInsertExecutor.java

```diff
@@ -61,6 +62,7 @@
 		Check.assumeNotNull(fromQuery, "fromQuery not null");
 		this.fromQuery = fromQuery;
 		this.fromModelClass = fromQuery.getModelClass();
+		this.fromTableName = InterfaceWrapperHelper.getTableName(fromModelClass);
```

Comment:
i can't comment on lines 54-58 but - boyscout-rule - please 
- annotate the two params as   `@NonNull`
- remove the Check.assumeNotNulls
- remove the super()

Reply:
done